cmd/spire-server/cli/localauthority/jwt/jwt_revoke_test.go

```diff
@@ -45,7 +45,7 @@ func TestJWTRevoke(t *testing.T) {
 				ExpiresAt:   1001,
 			},
 			expectStdoutPretty: "Revoked JWT authority:\n  Authority ID: revoked-id\n  Expires at: 1970-01-01 00:16:41 +0000 UTC\n",
-			expectStdoutJSON:   `{"revoked_authority":{"authority_id":"revoked-id","expires_at":"1001"}}`,
+			expectStdoutJSON:   `{"revoked_authority":{"authority_id":"revoked-id","expires_at":"1001","upstream_authority_subject_key_id":""}}`,
```

Comment:
why is `upstream_authority_subject_key_id` displayed here? may we omit empty?

Reply:
This is an issue in our `cliprinter` package. I filed https://github.com/spiffe/spire/issues/5523 for this.